src/System.Linq.Expressions/tests/SequenceTests/SequenceTests.cs

```diff
@@ -2960,29 +2929,6 @@ public static void InvokeQuotedLambda(bool useInterpreter)
             Assert.Equal(6, d());
         }
 
-        [Theory(Skip = "870811")]
```

Comment:
Would it matter here either way? And on the other side, how can we test something works in AOT within our tests?